test/napi.test.js

```diff
@@ -93,95 +93,109 @@ exports.test_list_networks = function (test) {
 
 
 exports.test_get_network = function (test) {
-    napi.getNetwork(NETWORKS[0].uuid, function (err, network) {
-        test.ifError(err);
-        test.ok(network);
-        test.equal(network.uuid, NETWORKS[0].uuid);
+    napi.getNetwork(ADMIN.uuid, function (err, network) {
+        test.ifError(err, 'getNetwork does not error');
+        test.ok(network, 'getNetwork returns a result');
+        test.ok(network.uuid, 'getNetwork result lacks a uuid');
+        if (network.uuid) {
+            test.strictEqual(network.uuid, ADMIN.uuid);
+        }
         test.done();
     });
 };
 
 
-// should fail since network doesn't belong to given owner
+// should fail since admin network doesn't belong to given owner
 exports.test_get_network_with_params = function (test) {
```

Comment:
Same here, would naming it `test_get_admin_network_with_params` make the purpose of this test clearer?